config/config.template.ini

```diff
@@ -494,17 +494,21 @@ SubD =
 
 poc_title = %DESCR%: %MSG%
 poc_text = %DATE% %TIME% - %DESCR%: %MSG%
+poc_ric = %DESCR%
 
 # Section for ZVEI
 # default prio for all ZVEI - except you specify it different
 zvei_prio = true
 zvei_title = Alarm: %ZVEI%
 zvei_text = %DATE% %TIME%: %ZVEI%
+# zvei_ric working at 6 characters or more
+zvei_ric = %DESCR%
```

Comment:
Was ist eine ZVEI RIC? Das widerspricht sich!

Reply:
Der Parameter wird später als Alarmierungs-RIC in divera genutzt. Ich kenne nur RICs von POC / Divera. Mit ZVEI habe ich bisher nie zu tun gehabt.
Ist die bessere Beschreibung: zvei_relais?

Reply:
in BW2 heißt es intern `zvei_id`
Würde ich bis ins Plugin so durchziehen um Verwirrung zu vermeiden.
Wenn es in Divera selbst anders heißt am besten daneben kommentieren, damit klar ist was dieser Wert auf der anderen Seite des API darstellt.

Reply:
Ist kommentiert und entsprechend eingetragen.